ToDoApp/ToDoApp/ToDoService.swift

```diff
@@ -9,12 +9,13 @@ import Foundation
 
 enum TaskError: Error {
     case unavailableTask
+    case unsavedTask
 }
 
-struct TaskModel: Identifiable {
-    let id: UUID = UUID()
-    var title: String
-    var deadline: Date
+struct TaskModel: Identifiable, Codable {
+    var id: UUID = UUID()
+    var title: String = ""
+    var deadline: Date = Date()
```

Comment:
what was the point of those changes? we should not be able to change the id as well as create a value with default values because this introduce weakness to our model domain and makes it possible to make errors since compiler would not be able to guard us from those errors

Reply:
Oh it's just a mistake while changing to test the new implementation, I've changed it back

Reply:
without changing let id to var id, it will give this warning "Immutable property will not be decoded because it is declared with an initial value which cannot be overwritten"

Reply:
In that case it it would be better to remove default value like this
`let id: UUID`